compiler/parser/src/tokenizer/token.rs

```diff
@@ -180,6 +182,7 @@ impl Token {
             Token::Char => sym::char,
             Token::Console => sym::console,
             Token::Const => sym::Const,
+            Token::Constant => sym::Const,
```

Comment:
Should this go to something like `sym::Constant`? Disclaimer: I haven't looked at the symbol code, so if my comment doesn't make sense, please ignore it.

Reply:
We're going to keep `const` and emit a warning I believe.

Reply:
This is supposed to match to sym::Constant whoops.